scripts/geo_metadata.py

```diff
@@ -0,0 +1,552 @@
+import argparse
+import anndata as ad
+import lattice
+import os
+import pandas as pd
+import shutil
+import sys
+import scanpy as sc
+import re
+import subprocess
+from urllib.parse import urljoin
+import requests
+import gspread
+import numpy as np
```

Comment:
I think these aren't needed
anndata
os
shutil
scanpy
re
subprocess
gspread

Reply:
Thanks! That is definitely a lot of unused libraries. I've removed them and also removed a line of code that converted type that wasn't necessary.